app/src/main/res/layout/fragment_search_results.xml

```diff
@@ -29,6 +29,7 @@
     </LinearLayout>
 
     <androidx.core.widget.NestedScrollView
+        android:id="@+id/searchResultsNestedScrollView"
```

Comment:
Can you please explain, how defining an id solve the issue?

Reply:
@liveHarshit Well, in short, this is Android View State Storing Mechanism (to retain an amount of information/state if we provide it an id). So for instance, if you're having an EditText and some text in it, I believe the text is also retained after screen rotation if we provide it with an id.

So this is also the way current EventsFragment restoring its position after coming back from EventDetailsFragment.